spyder/plugins/tests/test_editor_introspection.py

```diff
@@ -58,6 +58,7 @@ def get_spyder_pythonpath(*args):
     editor.introspector.plugin_manager.close()
 
 
+@pytest.mark.slow
 @pytest.mark.skipif(os.environ.get('CI', None) is not None,
                     reason="This test fails too much in the CI :(")
```

Comment:
Please remove this condition to see if it passes in our CIs because else this test wouldn't be run anywhere.

Reply:
Ah yes, silly me. I tested it previously with #6044 since it was failing locally too, but after adding huge ``qtbot.wait()``s I got it to pass, though still not perfectly consistently. I figured that had a good chance at fixing it on the CIs, but at the cost of a minimum of 30+ seconds added to every build in the best case; if it failed (which just happened sometimes maybe 1 out of 10 on my machine) and reran 3x, that would multiply to 90, and with 3 flaky runs, (which never seemed to change the outcome and just triple the runtime when it failed, so I dropped them in #6099 ) it would be around 5 extra minutes of build time worst case, before failing anyway. That said, doesn't hurt giving it a shot...